app/views/step_by_step_pages/internal_change_notes.html.erb

```diff
@@ -18,30 +18,51 @@
 
 <%= render 'shared/steps/page_title', links: links %>
 
+<%= render 'shared/steps/form_notice', message: notice, status: "success" %>
+
 <%= render 'shared/steps/heading',
   step_nav: @step_by_step_page.title,
-  action: 'Record a new change note or review prior changes'
+  action: 'Change notes'
 %>
 
-<%= render 'nav', step_by_step_page: @step_by_step_page, active: "change-notes" %>
+<%= render 'nav', step_by_step_page: @step_by_step_page, active: "internal-change-notes" %>
 
 <%
   left_col = "col-md-7"
   right_col = "col-md-5"
+  full_width = "col-md-12"
 %>
 
 <%= form_for(@internal_change_note, url: step_by_step_page_internal_change_notes_path) do |form| %>
   <%= render "shared/steps/form_errors", resource: @step_by_step_page %>
 
-  <div class="form-group row">
-    <div class="<%= left_col %>">
-      <%= form.text_area :description, required: true, class: "form-control", rows: 8 %>
+  <div class="row">
+    <div class="<%= full_width %>">
+      <details class="add-change-note--details">
+        <summary class="add-change-note--summary">
+          <span class="btn btn-primary">Add a change note</span>
+        </summary>      
+        <div class="form-group">
+          <%= form.text_area :description, required: true, class: "form-control", rows: 8 %>
+          <%= form.submit "Save change note", class: "btn btn-success" %>
+        </div>
+      </details>
     </div>
   </div>
-  <div class="form-group">
-    <%= form.submit "Save", class: "btn btn-primary" %>
-  </div>
-  <div class="form-group">
-    <%= link_to 'Cancel', @step_by_step_page %>
+  <div class="row">
+    <div class="<%= full_width %>">
+      <% @step_by_step_page.internal_change_notes.each do |note| %>
+        <details class="change-note--details">
+          <summary class="change-note--summary">
+            <%= note.created_at.to_datetime.strftime('%A, %d %B %Y at %H:%M %p') %>
```

Comment:
I'd consider moving this out of the view and perhaps add a method to the `InternalChangeNote` model that returns a formatted date.